kong_test.go

```diff
@@ -75,11 +76,53 @@ func TestKongAdminAPI_ReturnVersion(t *testing.T) {
 	assert.Nil(t, err)
 
 	all, err := io.ReadAll(get.Body)
-	root, err := ajson.Unmarshal(all)
-	result, err := root.JSONPath("$.headers.host")
-	value, err := result[0].GetString()
 	assert.Nil(t, err)
+
+	type JSONResponse struct {
+		StartedDateTime time.Time `json:"startedDateTime"`
+		ClientIPAddress string    `json:"clientIPAddress"`
+		Method          string    `json:"method"`
+		URL             string    `json:"url"`
+		HTTPVersion     string    `json:"httpVersion"`
+		Cookies         struct {
+		} `json:"cookies"`
+		Headers struct {
+			Host            string `json:"host"`
```

Comment:
You could keep only the Headers.Host field in the struct, but I left it for demo purpose :)